cdap-ui/app/cdap/components/OpsDashboard/RunsList/index.js

```diff
@@ -55,11 +77,12 @@ function renderBody(data) {
             >
               <div>{run.namespace}</div>
               <div>{run.application.name}</div>
-              <div>{run.type}</div>
+              <div>{T.translate(`commons.entity.${run.type.toLowerCase()}.singular`)}</div>
```

Comment:
make sure you test for all the program types for this. A little concerned about whether the backend returns a singular form consistently or not

Reply:
Have checked with Mao, the backend will always return singular form

Reply:
I'd still do a manual test for all the programs. Should be simple to test

Reply:
Have manually checked all the program types, except Worker, since I couldn't find a sample program of this type